### PR TITLE
RELATED: BB-1112 Support arithmetic measure in UI SDK

### PR DESCRIPTION
**Pull request by xMort**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/100/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/116/ : SUCCESS in 39s

File: src/DataLayer/utils/AfmUtils.ts [hunk 1]
```diff
@@ -94,6 +105,17 @@ export function isPreviousPeriodMeasure(item: AFM.IMeasure): boolean {
...
 }
 
+/**
+ * Returns true if measure is arithmetic measure
+ *
+ * @method isArithmeticMeasure
+ * @param {AFM.IMeasure} item
+ * @returns {boolean}
+ */
+export function isArithmeticMeasure(item: AFM.IMeasure): boolean {
```
[the remark this answers is not in the document]
Comment (xMort) [reply src/DataLayer/utils/AfmUtils.ts]: Unfortunately, it cannot be converted to type guard as `IMeasure` is not `type` but `interface`. The type that is being tested is the `definition` inside of the object :(

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/101/ : SUCCESS in 39s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/117/ : SUCCESS in 2m 50s

**Comment by tomassatinsky** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/35/ : SUCCESS in 45s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/118/ : SUCCESS in 1m 13s

**Pull request merged**